src/rgw/rgw_rados.h

```diff
@@ -912,7 +912,7 @@ class RGWRados
 
       int init_bs(const DoutPrefixProvider *dpp) {
         int r =
-	  bs.init(target->get_bucket(), obj, nullptr /* no RGWBucketInfo */, dpp);
+	  bs.init(target->get_bucket(), obj, &target->bucket_info, dpp);
```

Comment:
the change itself looks correct, but i don't think the description is accurate - we're not updating the bucket info here, we're asking `init()` to use our existing info instead of fetching updated info from cache/rados